### DISCUSSION TITLE
「闷声发财」浏览器通知实现

### DISCUSSION THREAD

**Discussion by skyzh** [skyzh/make-a-fortune#36] [Ideas]
- 开一个 WebWorker 10min pull 一次消息
- 如果有新的消息，发通知

cc @keithnull

**Comment by izackwu** [comment]
听上去好像不难？但是我没写过 web notification，所以不太确定

**Reply by skyzh** [reply]
感觉上应该也不难。但是取设置之类的操作需要 React 的 runtime，不确定 WebWorker 里能不能做（

**Reply by PhotonQuantum** [reply]
看了一下，webworker 是直接读不到 localstorage 和 sessionstorage 的，需要用 IndexedDB

**Reply by skyzh** [reply]
可以考虑每次前端更新 token 的时候（或打开任意页面的时候）都把 Token post 到 worker
